packages/server-test-utils/dist/vue-server-test-utils.js

```diff
@@ -19,14 +19,12 @@ function createVNodesForSlot (
   slotValue,
   name
 ) {
-  if (typeof slotValue === 'string' &&
-  !startsWithTag(slotValue)) {
+  if (typeof slotValue === 'string' && !startsWithTag(slotValue)) {
```

Comment:
These changes are not related to the issue, files are automatically updated on `yarn test` or something, but they shouldn't be included in the PR

Reply:
Whoops! Thanks :relaxed: